envoy/server/access_log_config.h

```diff
@@ -1,54 +0,0 @@
-#pragma once
-
-#include <string>
-
-#include "envoy/access_log/access_log.h"
-#include "envoy/config/typed_config.h"
-#include "envoy/server/filter_config.h"
-
-#include "source/common/protobuf/protobuf.h"
-
-namespace Envoy {
-namespace Server {
-namespace Configuration {
-
-/**
- * Implemented for each AccessLog::Instance and registered via Registry::registerFactory or the
- * convenience class RegisterFactory.
- */
-class AccessLogInstanceFactory : public Config::TypedFactory {
```

Comment:
Why is this Factory being deleted? Is it just being moved? If so, why is it being moved? Maybe update the PR description to explain.

Reply:
See https://github.com/envoyproxy/envoy/pull/30734/files#r1384286231

This factory will be replaced by AccessLogInstanceFactoryBase<HttpLogContext>